data-url/tests/wpt.rs

```diff
@@ -135,6 +135,7 @@ fn collect_mime<F>(add_test: &mut F)
 }
 
 fn main() {
+    std::panic::set_hook(Box::new(|_| {}));
```

Comment:
I don’ think this is appropriate. Panic output/backtraces can be essential, especially when tests are failing.

Reply:
Addressed by https://github.com/servo/rust-url/pull/473/commits/d593f4ca03e12d4be19836040d91a533aca1daa7